src/expr/node_value.cpp

```diff
@@ -37,16 +37,16 @@ namespace expr {
 string NodeValue::toString() const {
   stringstream ss;
 
-  OutputLanguage outlang = (this == &null()) ? language::output::LANG_AUTO
-                                             : options::outputLanguage();
+  Language outlang =
+      (this == &null()) ? Language::LANG_AUTO : options::outputLanguage();
```

Comment:
Not for this PR, but maybe we should introduce `NodeValue::isNull()`. This is just ugly.